tests/themes-project/themes/blog_theme/theme.ini

```diff
@@ -0,0 +1,16 @@
+name = "Blog theme"
+license = "MIT"
+licenselink = ""
+description = "And Example blog theme for testing"
+homepage = ""
+tags = "simple, minimal, blog"
+features = "blog"
+lektor_required_version = 3.1
```

Comment:
Maybe this should be `lektor_minimum_required_version`, since it's unclear if we want to lock it to a particular version or require at least the given version. If it should remain as `lektor_required_version` I think it should support the same quantifiers that can be used in a `setup.py` (`>=3.0,<4`)

Reply:
Yes, I agree, I'll change it to lektor_minimum_required_version, I don't think that supporting maximum or pinned versions is a good idea

Reply:
:+1: